src/Stratis.Bitcoin/Configuration/TextFileConfiguration.cs

```diff
@@ -116,15 +124,11 @@ public void MergeInto(TextFileConfiguration destination)
         /// <returns>Values for the specified argument.</returns>
         public string[] GetAll(string key)
         {
-            // Get the values without the - prefix.
-            if (!this.args.TryGetValue(key, out var values))
-                values = new List<string>();
-
             // Get the values with the - prefix.
-            if (!this.args.TryGetValue($"-{key}", out var dashValues))
-                dashValues = new List<string>();
+            if (!this.args.TryGetValue($"-{key}", out var values))
```

Comment:
I think this will be a problem if someone calls GetAll with a key containing a dash.
It's hard to tell the user to not use a dash here so support for both was quite important I think.

Reply:
We don't have any current cases of passing dash-prefixed keys. I will update the documentation to make it clear that non-dash prefixed keys should be used when calling GetAll.

Reply:
yeah I completely agree that it should be uniform, whichever one we choose.